### PR TITLE
Add info section to Splash component

### PR DESCRIPTION
**Pull request by robines**
Currently you can’t interact with the splash component in any way, which may be annoying for users. This PR adds the event's short description, a "More info" button, and if the event is paid: a "Buy ticket" button. The "Buy ticket" button does nothing currently.

I think another solution is required for mobile, so currently the info section is hidden for mobile. Perhaps just making the splash image clickable for mobile?

Screenshot:
screenshot

# Notable changes

- Add `PAID_TICKET_TYPES` constant
- Add `link` property to Button component. If non-empty, will switch out `button` element with Link component.

### CODE REVIEW

File: frontend/src/Pages/HomePage/components/Splash/Splash.tsx [hunk 1]
```diff
@@ -20,6 +30,30 @@ export function Splash({ events }: SplashProps) {
   const imageUrl = events ? BACKEND_DOMAIN + events[index].image_url : '';
   const nextImageUrl = events ? BACKEND_DOMAIN + events[nextIndex].image_url : '';
 
+  const description = events ? dbT(events[index], 'description_short') : '';
+
+  const ticketButton =
+    events && PAID_TICKET_TYPES.includes(events[index].ticket_type) ? (
```
Comment (emilte) [inline frontend/src/Pages/HomePage/components/Splash/Splash.tsx:36]: Kan flyttes ut til const

File: frontend/src/types.ts [hunk 2]
```diff
@@ -89,3 +89,4 @@ export type EventAgeRestriction = 'none' | 'eighteen' | 'twenty' | 'mixed';
 export type EventStatus = 'active' | 'cancelled' | 'archived' | 'deleted';
 
 export const ALL_TICKET_TYPES: EventTicketType[] = ['free', 'included', 'billig', 'registration', 'custom'];
+export const PAID_TICKET_TYPES: EventTicketType[] = ['billig', 'custom', 'registration'];
```
Comment (emilte) [inline frontend/src/types.ts:92]: Jeg tror disse bør bygges opp av konstanter for å sørge for at de er like

**Review by emilte — APPROVED** [review]

File: frontend/src/Pages/HomePage/components/Splash/Splash.tsx (hunk 1 above)
Comment (robines) [reply frontend/src/Pages/HomePage/components/Splash/Splash.tsx]: Regner med at du mener om arrangementet er betalt? Flyttet ut til `isPaid` i siste commit

**Review by emilte — APPROVED** [review]
Fint

File: frontend/src/Pages/HomePage/components/Splash/Splash.tsx (hunk 1 above)
Comment (emilte) [reply frontend/src/Pages/HomePage/components/Splash/Splash.tsx]: jep

**Pull request merged**